apps/speed.c

```diff
@@ -1533,17 +1491,19 @@ int speed_main(int argc, char **argv)
                 BIO_printf(bio_err, "%s: -evp option cannot be used more than once\n", prog);
                 goto opterr;
             }
-            evp_cipher = obtain_cipher(opt_arg(), &fetched_cipher);
-            if (evp_cipher == NULL) {
+            ERR_set_mark();
```

Comment:
Why?

Reply:
Because the -evp argument can be either cipher or md. If it is an md we need to clear the errors from the failed attempt of fetching a cipher.